saleor/webhook/error_codes.py

```diff
@@ -8,3 +8,10 @@ class WebhookErrorCode(Enum):
     REQUIRED = "required"
     UNIQUE = "unique"
     DELETE_FAILED = "delete_failed"
+
+
+class WebhookDryRunErrorCode(Enum):
+    GRAPHQL_ERROR = "graphql_error"
+    NOT_FOUND = "not_found"
+    INVALID_ID = "invalid_id"
+    MISSING_PERMISSION = "missing_app_permission"
```

Comment:
`missing_app_permission`? Is it correct? I thought that the dry run is called by the staff user, without using any existing app.

Reply:
Good spot! Thanks